style.css

```diff
@@ -0,0 +1,192 @@
+html, body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+.content {
+  flex: 1 0 auto;
+}
+
+.site-header {
+  background-color: #F2B4F4;
+}
+
+.site-main {
+  background-color: #F4F4F4;
+}
+
+.site-footer {
+  background-color: #072BCE;
+  color: #FFFFFF;
+  text-align: center;
+  flex-shrink: 0;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.first-section {
+  background-color: #F4F4F4;
+  border: 4px solid red;
+}
+
+.second-section {
+  border: 4px solid red;
+}
+
+.project-list {
+  padding: 0;
+  gap: 25px;
+}
+
+.project-list-item {
+  border: 2px solid #000000;
+}
+
+.feature1 {
```

Comment:
why `.feature1`? why not just `.feature`?

Reply:
Fixed.